test/tests/kernels/simple_diffusion/simple_diffusion.i

```diff
@@ -1,8 +1,8 @@
 [Mesh]
   type = GeneratedMesh
-  dim = 2
-  nx = 10
-  ny = 10
+  dim = 1
+  nx = 2
+  # ny = 2
```

Comment:
damn it this is what I get for trying to be funny @fdkong

Reply:
Haha